nodeup/pkg/model/kubectl.go

```diff
@@ -106,8 +106,12 @@ func (b *KubectlBuilder) findKubeconfigUser() (*fi.User, *fi.Group, error) {
 	switch b.Distribution {
 	case distros.DistributionJessie, distros.DistributionDebian9, distros.DistributionDebian10:
 		users = []string{"admin", "root"}
-	case distros.DistributionCentos7:
+	case distros.DistributionXenial, distros.DistributionBionic, distros.DistributionFocal:
+		users = []string{"ubuntu"}
+	case distros.DistributionCentos7, distros.DistributionCentos8:
 		users = []string{"centos"}
+	case distros.DistributionAmazonLinux2, distros.DistributionRhel7, distros.DistributionRhel8:
+		users = []string{"ec2-user"}
```

Comment:
Hm this logic is used on every cloud provider. AL2 makes sense to use ec2-user but RHEL7 and RHEL8 might have different users depending on the cloud provider. Maybe we add a conditional for `aws` here? though it would be good to cover other cloud providers as well.

Reply:
I think it has multiple choices, so should be ok. Maybe someone knows what to add for other cloud providers for centos.

Reply:
ah you're right. we can encourage users of other cloud providers to submit additions in separate PRs